hikari/impl/rest.py

```diff
@@ -717,39 +718,6 @@ async def _parse_ratelimits(self, compiled_route: routes.CompiledRoute, response
             retry_after=body_retry_after,
         )
 
-    @staticmethod
-    @typing.final
-    def _generate_allowed_mentions(
-        mentions_everyone: undefined.UndefinedOr[bool],
-        mentions_reply: undefined.UndefinedOr[bool],
-        user_mentions: undefined.UndefinedOr[typing.Union[snowflakes.SnowflakeishSequence[users.PartialUser], bool]],
-        role_mentions: undefined.UndefinedOr[typing.Union[snowflakes.SnowflakeishSequence[guilds.PartialRole], bool]],
-    ) -> data_binding.JSONObject:
-        parsed_mentions: typing.List[str] = []
-        allowed_mentions: typing.Dict[str, typing.Any] = {"parse": parsed_mentions}
-
-        if mentions_everyone is True:
-            parsed_mentions.append("everyone")
-
-        if mentions_reply is True:
-            allowed_mentions["replied_user"] = True
-
-        if user_mentions is True:
-            parsed_mentions.append("users")
-        elif isinstance(user_mentions, typing.Collection):
-            # Duplicates will cause discord to error.
-            ids = {str(int(u)) for u in user_mentions}
-            allowed_mentions["users"] = list(ids)
-
-        if role_mentions is True:
-            parsed_mentions.append("roles")
-        elif isinstance(role_mentions, typing.Collection):
-            # Duplicates will cause discord to error.
-            ids = {str(int(r)) for r in role_mentions}
-            allowed_mentions["roles"] = list(ids)
-
-        return allowed_mentions
-
```

Comment:
This makes no sense to be moved to `special_endpoints`, but rather to a `internal` class

Reply:
I mean i just moved it from one impl module to another but if that's what you say then whatever